starboard/build/util/generate_runfile.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""
+Generate runfiles for evergreen.
+"""
+import os
+import stat
+import sys
+
+_TEMPLATE = """#!/usr/bin/env python3
+import subprocess
+import sys
+
+command = [
+    '{}/elf_loader_sandbox', '--evergreen_content=.',
+    '--evergreen_library={}.so'
+] + sys.argv[1:]
+result = subprocess.run(command, check=False)
```

Comment:
Add `capture_output = True`? Otherwise there will be no output from the command.

From [docs.python.org](https://docs.python.org/3/library/subprocess.html#subprocess.run):

```
If capture_output is true, stdout and stderr will be captured. When used, the internal Popen object is 
automatically created with stdout and stderr both set to PIPE. The stdout and stderr arguments may not 
be supplied at the same time as capture_output. If you wish to capture and combine both streams into 
one, set stdout to PIPE and stderr to STDOUT, instead of using capture_output.
```

Reply:
We don't want to capture the output, we want it to be output to the console as if we were running the command directly from the command line